src/main/charts/jira/values.yaml

```diff
@@ -533,6 +542,24 @@ jira:
     #
     localHomeSubPath: "log"
 
+  # Use AWS S3 to store attachments. From Confluence 8.1 onwards.
+  #
+
+  s3AvatarsStorage:
```

Comment:
Just wondering what happens when attachments for s3 needs to be added? Do we create a new stanza like this i.e.

```
s3AttachmentStorage
  bucketName
  bucketRegion
  endpointOverride
```

Or is it better to just rename this stanza to `s3Storage` and its used when storing avatars or attachments or both?

Reply:
I think it is possible that different buckets may be used for avatars and attachments. Maybe:

```
s3Storage:
  avatars:
    bucketName
    bucketRegion
    endpointOverride
  attachments:
    bucketName
    bucketRegion
    endpointOverride
```

Reply:
Oh interesting. Is this an option @asezavar?

Reply:
Of course, if we expect attachments to arrive soon :)

Reply:
@errcode1202 yes that's the plan

Reply:
Should the stanza be updated now then to accomadate attachments?

Reply:
I just changed the s3 storage values structure based on your suggestion, but I haven't added the attachment part, and there will be another ticket for that.